earthpy/tests/test_spatial.py

```diff
@@ -1,5 +1,88 @@
 from earthpy import spatial
+from osgeo import gdal
+from osgeo import osr
+import rasterio as rio
+import numpy as np
 
+# a helper function to generate a 3D array of data
+def createArray(arr_shape):
```

Comment:
@joemcglinchy this looks great! I wonder whether we can remove the `createArray` function, since I think we will want to be able to make more types of dummy arrays than what this currently implements, and we can do that pretty easily using `np.ones()` one-liners + the assertion that you wrote in `dummyTifWriter`. For example, if we need to make arrays that have weird bounds to test byte scaling, or arrays with negative values we can do that with `np.random.uniform(..., size=(rows, cols, channels))` or `np.ones((rows, cols, channels)) * -1`.

Reply:
i'm sorry to muck up the convo here with my syntax stickling ... but can we please follow PEP8 guidelines

https://www.python.org/dev/peps/pep-0008/#function-and-variable-names

Rather than camel case i prefer all `lower_case_with_underscore` naming conventions. Please yell if this is not in line with your thoughts on this package.

Reply:
oops just noticed this actually is still using camelCase rather than snake_case. camel being reserved for class names only. can you kindly fix @joemcglinchy